.gitignore

```diff
@@ -6,4 +6,10 @@ dist/
 config.js
 
 
-coverage
+coverage/
+
+__snapshots__/
+
+test.png
+
+__tests__/overview/puppeteer.test.js
```

Comment:
just confirm if you really mean to ignore this test case.

Reply:
Yes, I will include it in a later commit!